hbase-commit-table/src/main/java/com/yahoo/omid/committable/hbase/HBaseCommitTable.java

```diff
@@ -86,9 +87,11 @@ protected HBaseCommitTable(Configuration hbaseConfig, String tableName, KeyGener
 
     public class HBaseWriter implements Writer {
 
+        private static final long INITIAL_LWM_VALUE = -1L;
         final HTable table;
         // Our own buffer for operations
         final List<Put> writeBuffer = new LinkedList<>();
+        final AtomicLong lowWatermarkToStore = new AtomicLong(INITIAL_LWM_VALUE);
```

Comment:
This is another reason to get ride of the reset and allocate a new object at failover. 
This is because currently we are using atomic long in the mainstream execution only for races that can appear during a rare failover.

Besides, everything looks good.